overlays/bootstrap.nix

```diff
@@ -164,6 +164,7 @@ in {
                 ++ final.lib.optional (versionAtLeast "8.10.3" && final.targetPlatform.isAarch64) ./patches/ghc/3434.patch
 
                 ++ from      "8.10.1"          ./patches/ghc/ghc-acrt-iob-func.patch
+                ++ from      "8.10.7"          ./patches/ghc/ghc-mprotect-nonzero-len.patch
```

Comment:
I think this problem extends back to 8.10.1, which was the first release to have `mmapForLinkerMarkExecutable`.

Reply:
fair point. I'll push a revised one after a test bake tomorrow

Reply:
okay, done